specification/maps/data-plane/Search/preview/2022-09-01-preview/geocoding.json

```
@@ -0,0 +1,1074 @@
+        "operationId": "Search_PostGeocodingBatch",
+        "x-ms-examples": {
+          "A Geocoding Batch API call containing 2 Geocoding queries": {
+            "$ref": "./examples/ForwardGeocodingBatch.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          },
+          {
+            "name": "geocodingBatchRequestBody",
+            "in": "body",
+            "description": "The list of address geocoding queries/requests to process. The list can contain a max of 100 queries and must contain at least 1 query.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchRequestBody"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchResponse"
+            }
+          },
+          "207": {
+            "description": "Multi-Status. One or more of the batch items could not be processed due to error. Items successfully processed but with no results are not considered partial content.",
```

Since partial content is removed in the status code, the "partial content" in the sentence may not be cleared. Could we change it to something similar to the suggestion? 

```suggestion
            "description": "Multi-Status. One or more of the batch items could not be processed due to error. Items successfully processed but with no results are not considered as error.",
```

---

I have just updated the whole description. Please let me know if it doesn't work for you.